MODEL-GUIDE.md

```diff
@@ -118,3 +118,119 @@ The build script will enforce the following keyword conventions.
     ```
 
 
+## x-pattern extension
+### schema
+```yaml
+x-pattern: 
+  description: >-
```

Comment:
Where are we documenting what `pattern` means in our spec ? could we refer to that in this description ?

Reply:
This is the documentation for the x-pattern extension. Any extension should be described as a schema object.

Reply:
Should we have a separate doc for extensions or is the model guide ok?